src/config/logger.ts

```diff
@@ -35,7 +35,7 @@ const jobSerializer = (job: Job) => job && {
 	...job,
 	// Removing potentially extremely large amount of data from logs
 	data: undefined,
-	queue: undefined
+	queueName: job?.queue?.name
```

Comment:
Please don't change this. We'll have to get used to new splunk queries format

Reply:
Or, hold on, is it a duplicating parameter within { job.queueName } ?

Reply:
Ok, never mind. Hence Redis queues going away anyway. 

I am happy as long as we have an original "queue" parameter in splunk messages (which is queue name)